bin/si-mcp-server/src/tools/schemaCreate.ts

```diff
@@ -11,32 +10,56 @@ import {
 } from "./commonBehavior.ts";
 
 const name = "schema-create";
-const title = "Create a new Schema";
-const description =
-  `<description></description><usage></usage>`;
+const title = "Create a new schema";
+const description = `
+<description>
+Create a new schema, which can then be used to create components of that type.
+</description>
+<usage>
+</usage>
+`;
 
-const SchemaCreateInputSchemaRaw = {
-  changeSetId: z.string().describe(
-    "The change set to create a schema in; schemas cannot be created on HEAD",
-  ),
+const DEFAULT_SCHEMA_DEFINITION_FUNCTION = `function main() {
+    const asset = new AssetBuilder();
+    return asset.build();
+}`;
+
+const InputSchemaRaw = {
```

Comment:
We usually call these types after their tool e.g. what it was before - that's the convention of the app